### ISSUE TITLE
Question about masked language model 

### ISSUE THREAD

**Issue by paprika0741** [649453932/Bert-Chinese-Text-Classification-Pytorch#176]
代码中masked language modeling labels中-1标记的是被masked的token，loss计算忽略被mask的token，但是BERT论文中写的是”the final hidden vectors corresponding to the mask tokens are fed into an output softmax over the vocabulary“ 只计算masked token处的loss
